<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorWgt</class>
 <widget class="QWidget" name="SensorWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="7,3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ultrasound Sensor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Connected</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Warning Distance(mm)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="ShowStatusLEDUTSensor1">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>UT Sensor 1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="EnableUTSensor1">
          <property name="text">
           <string>CheckBox</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="WarningValueUTSensor1">
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="value">
           <number>300</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="ShowStatusLEDUTSensor2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>UT Sensor 2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="EnableUTSensor2">
          <property name="text">
           <string>CheckBox</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="WarningValueUTSensor2">
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="value">
           <number>300</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="2,6">
        <item>
         <widget class="QPushButton" name="refreshUTSenParamBtn">
          <property name="text">
           <string>refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Distance1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="showCurDistance1">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="showUTStatus1">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Distance2:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="showCurDistance2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="showUTStatus2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Pressure Sensor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Connected</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Warning Pressure Value</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="ShowStatusLEDPreSensor1">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Pressure Sensor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>CheckBox</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="WarningValuePreSensor1">
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>2000</number>
            </property>
            <property name="value">
             <number>600</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="2,6">
        <item>
         <widget class="QPushButton" name="refreshPreSenParamBtn">
          <property name="text">
           <string>refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Pressure1:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="showCurPressure1">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Status:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="showPreStatus1">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
